presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraClientModuleTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cassandra;
+
+import com.datastax.driver.core.Cluster;
+import com.facebook.airlift.json.JsonCodec;
+import org.mockito.Answers;
+import org.mockito.Mockito;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+public class CassandraClientModuleTest {
```

Comment:
Our convention is `Test` is first, i.e. `TestCassandraClientModule`.  Also, it's not typical to test Guice modules, as this is more or less configuration.  Would it be possible to create an end to end test that proves the configuration is being used?